system/bin/chroot-distro

```diff
@@ -561,23 +561,27 @@ chroot_distro_download() {
                 1) rootfs="full";rootfs_version="kali-2024.1"; break;;
```

Comment:
Are you aware that Kali Linux does not provide 2024.1 version of nethunter anymore? Thus 1-3 should be removed.

Reply:
Sure, but it isn't at the same location as the others.

Reply:
Also, since it is archive then I would assume that it isn't currently actively supported. So, there is no need to keep them supported by ourselves.